src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -30,6 +30,11 @@ public BlazorWebView()
 		/// </summary>
 		public string? HostPage { get; set; }
 
+		/// <summary>
+		/// Gets or sets the path for initial navigation when the application starts.
```

Comment:
```suggestion
		/// Gets or sets the path for initial navigation within the Blazor navigation context when the Blazor component is finished loading.
```

^^ I wanted to be as clear as possible about exactly what path this affects, so maybe this helps?